### DISCUSSION TITLE
Filer metadata events

### DISCUSSION THREAD

**Discussion by amir-latifi** [seaweedfs/seaweedfs#5837] [Q&A]
Hi,
I started a master, 3 volumes with "ssd" disk and a filer with disk flag set to "ssd". Uploading files are completely fine. But i have to many errors in filer about failure of writing metadata event with this description "no free volumes left". 

I think filer can write topics just on "hdd" disk. Am I write? Isn't it better to write topic logs on the same disk type which filer has? 
Is there anyway other than fs.configure to change disk type of topics logs?

**Comment by chrislusf** [comment]
if all the disk type is "ssd", you can leave disk type as empty.
